src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/IndexConfiguration/BulkIndexationIntegration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\Elasticsearch\IndexConfiguration;
```

Comment:
I moved this test on Akeneo namespace and added tests on save() & saveAll() product saver to check if products are indexed